packages/api/src/models/cropVarietyModel.js

```diff
@@ -83,8 +83,8 @@ class CropVariety extends BaseModel {
         crop_id: { type: 'integer' },
         farm_id: { type: 'string' },
         crop_variety_name: { type: ['string', null] },
```

Comment:
Not sure i understand what you're getting at, could you elaborate?

Reply:
I just thought `crop_variety_name` type should not be `null` since it is a [required field](https://github.com/LiteFarmOrg/LiteFarm/blob/93460b3db68f4b920ae0bebb6c128fff139ede6f/packages/webapp/src/components/AddCropVariety/index.jsx#L54) on the frontend.
But I guess it had been like this and is better not to change... never mind!

Reply:
Oh that makes sense. But I didn't actually add or change that line of code and don't think any changes to it are expressly required within the scope of this ticket